### DISCUSSION TITLE
yarn & uikit

### DISCUSSION THREAD

**Discussion by mrsaoc** [balta-io/forum#459] [Tira Dúvidas] [answered]
Boa noite, estou no curso: 7181 módulo 2 na aula de compilação de tema.

instalo o yarn (1.22.21) e copio o passo-a-passo do site uikit para adiciona-lo, porém não consigo compilar. Sabe o que possa ser?

![image](https://github.com/balta-io/forum/assets/61404851/c3fa184f-fb64-4b8b-a2f5-c92bd69d6053)

**Comment by andrebaltieri** [comment, marked as answer]
Bom dia, @marcosvmg, como vai?

Acredito que a versão do Node que está usando, esteja diferente da apresentada no curso, tanto ele quanto o NPM devem corresponder as mesmas versões apresentadas nas aulas.
